rosdep/base.yaml

```diff
@@ -342,6 +342,8 @@ box2d-dev:
   debian: [libbox2d-dev]
   fedora: [Box2D-devel]
   ubuntu: [libbox2d-dev]
+build-essential:
+  ubuntu: [build-essential]
```

Comment:
We've never needed `build-essential` as a rosdep rule before, can you explain the scenario where you needed this? I think it basically gets pulled in by our other low level dependencies, though maybe that's an oversight on our part. Also this probably won't translate into anything meaningful on other OS's.